src/legacy/server/config/schema.test.js

```diff
@@ -101,128 +101,6 @@ describe('Config schema', function () {
       });
     });
 
-    describe('ssl', function () {
```

Comment:
I assume parallel tests exist in the NP already?

Reply:
yup. https://github.com/restrry/kibana/blob/3a831a91a66c6b3d4dae41fc43af62e2593fbb0d/src/core/server/http/http_config.test.ts#L77